storage/tools/dump_tree/dumplib.go

```diff
@@ -474,9 +474,9 @@ func dumpLeaves(ctx context.Context, ls storage.LogStorage, tree *trillian.Tree,
 	}()
 
 	for l := int64(0); l < int64(ts); l++ {
```

Comment:
Consider squashing this to one `GetLeavesByRange` call (in this or separate PR).

Reply:
I'll do this in a separate PR.

Reply:
I'll do this in a separate PR.